app/main.py

```diff
@@ -1,6 +1,18 @@
-from typing import Callable
+from typing import Callable, Any
 
 
 def cache(func: Callable) -> Callable:
-    # Write your code here
-    pass
+    cache_storage = {}
+
+    def inner(*args, **kwargs) -> Any:
```

Comment:
Add pls type hint for `args` , and also you do not need here kwargs ( explained below )